### PR TITLE
Image node UI improved during image loads

### PR DESCRIPTION
**Pull request by ShavidzeT**
fix issue #181

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -35,3 +35,27 @@ h1 {
 .pr-item {
   word-break: break-word;
 }
+
+.u-bgc--f4f4f4 {
+  background-color: #f4f4f4;
+}
+.u-overflow--hidden {
+  overflow: hidden;
```
Comment (jenkoian) [inline public/css/main.css]: tachyone already has a `overflow-hidden` class we can use.

File: public/css/main.css [hunk 2]
```diff
@@ -35,3 +35,27 @@ h1 {
...
+}
+.u-opacity--disable {
+  opacity: 0;
+}
+.u-opacity--enable {
+  opacity: 1;
+}
```
Comment (jenkoian) [inline public/css/main.css]: Tachyons already has `o-100` for this.

File: public/css/main.css [hunk 3]
```diff
@@ -35,3 +35,27 @@ h1 {
...
+
+.u-bgc--f4f4f4 {
+  background-color: #f4f4f4;
+}
+.u-overflow--hidden {
+  overflow: hidden;
+}
+.u-opacity--disable {
+  opacity: 0;
+}
```
Comment (jenkoian) [inline public/css/main.css]: Tachyons already has `o-0` for this.

File: public/css/main.css [hunk 4]
```diff
@@ -35,3 +35,27 @@ h1 {
...
+.u-opacity--enable {
+  opacity: 1;
+}
+
+.Image {
+  width: auto;
+}
```
Comment (jenkoian) [inline public/css/main.css]: Use tachyons `w-auto`

File: public/css/main.css [hunk 5]
```diff
@@ -35,3 +35,27 @@ h1 {
...
+.Image--small {
+  width: 128px;
+  height: 128px;
+}
```
Comment (jenkoian) [inline public/css/main.css]: Can you use some of the existing width/height classes https://github.com/tachyons-css/tachyons-widths https://github.com/tachyons-css/tachyons-heights ?

File: public/css/main.css [hunk 6]
```diff
@@ -35,3 +35,27 @@ h1 {
...
+.Image--transition {
+  transition: all 1s ease;
+}
```
Comment (jenkoian) [inline public/css/main.css]: Does the tachyons class `glow` fit for this?

**Comment by ShavidzeT** [comment]
I'm glad to see your comments in my PR.
Will refactor to tachyons utility classes. 💯 🌞

**Comment by ShavidzeT** [comment]
@jenkoian refactored and completely adapted on tachyone's utils.
Let's review latest commits to fix image ui problem on production... ✌️

**Comment by jenkoian** [comment]
Thanks @ShavidzeT !

**Pull request merged**